pkg/service/test/test.go

```diff
@@ -465,17 +466,39 @@ func (t *tester) testHttp(tc models.TestCase, actualResponse *models.HttpResp) (
 	noise := tc.Noise
 
 	var (
-		bodyNoise   []string
-		headerNoise = map[string]string{}
+		bodyNoise   map[string][]string
+		headerNoise map[string][]string
```

Comment:
why this change form map[string]string{} -> map[string][]string{} has been made?

Reply:
earlier headerNoise was serving the purpose as a set, now it should accommodate the regex expressions given by the user for a noisy field.